Packs/FeedMicrosoftIntune/TestPlaybooks/playbook-FeedMicrosoftIntune_Test.yml

```
@@ -0,0 +1,95 @@
+id: playbook-FeedMicrosoftIntune_Test
+name: playbook-FeedMicrosoftIntune_Test
+version: -1
+fromversion: 4.5.0
```

```suggestion
fromversion: 5.5.0
```
The integration is only from server version 5.5.0 so the test playbook should match.